README.md

```diff
@@ -209,8 +209,7 @@ Markers and other elements need to be recomposed in the screen. To achieve recom
 have a mutable state variable:
```

Comment:
Perhaps say "To achieve recomposition you can set mutable properties of state objects:"